blog/EclipseFuseTooling-NewGlobalConfiguration.adoc

```diff
@@ -0,0 +1,30 @@
+

```

Comment:
just noticed this - please try not to use mixed cases in docs used for urls. all lower case please ;)